front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -205,6 +209,14 @@ export const VaultDataSourceViewContentList = ({
     [setPagination, setViewType, viewType, pagination.pageSize]
   );
 
+  const isServerPagination =
```

Comment:
for managed data source we won't get the search then?

Reply:
Yes,  managed data sources are not paginated serverside, all data loaded in the table, so search works